### PR TITLE
validate SpaceX

### PR DESCRIPTION
**Pull request by dorudil2**
# Validate [SPX](https://solscan.io/token/6mjbu9CpcJ1LdxXK9U9gzrXZ2K22D1xBTTUfsVzLh4gB)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/dorel_grig69884/status/1802759030932172888
- Coingecko/ CMC URL (If available):

## Validation (Please check off boxes):
- [X] The metadata provided in the PR matches what is on-chain (Mandatory)
- [X] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [X] My change matches the format in the file (no spaces between fields).
- [X] My token is already live and trading on Jupiter.
- [X] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [X] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Review by dorudil — COMMENTED** [review]
cool

File: validated-tokens.csv [hunk 1]
```diff
@@ -1494,3 +1494,4 @@ Jum Cromer,JUM,43eeGP8k9AZKBSyd9iahhThNtE3vcZvBWMaHNd2akM5s,6,https://bafybeicd3
 iceSOL,iceSOL,iceSdwqztAQFuH6En49HWwMxwthKMnGzLFQcMN3Bqhj,9,https://xsbf3mqssacc6xatyv27ornaiylcu5rw7qpbkciazgh6xwbpabja.arweave.net/vIJdshKQBC9cE8V190WgRhYqdjb8HhUJAMmP69gvAFI,true
 Nayeb Bekele,Bekele,4pPMX85G5t6M8hoKvdsbRDGbMkXxRg1vefcQ7MVjjMKM,6,https://bafkreiaphyug77bsxdnx55jmzppii3ttgaenxykz6kedlgowoun7ha53ku.ipfs.nftstorage.link,true
 Worken,WORK,9tnkusLJaycWpkzojAk5jmxkdkxBHRkFNVSsa7tPUgLb,5,https://ipfs.io/ipfs/Qmb9qSYq9YNMg3gejXnN9pstauXEr5czRc1rpgBhEKPnxm,true
+SpaceX,SPX,6mjbu9CpcJ1LdxXK9U9gzrXZ2K22D1xBTTUfsVzLh4gB,6,https://bafybeic2orrdeiyzm47kscnwi5mvu5a53qzpwq3zhxstoxcpnkdor5qvlu.ipfs.nftstorage.link,true
```
Comment (dorudil) [inline validated-tokens.csv:1497]: coll

**Comment by Jymfletzfur** [comment]
insufficient volume

**Comment by dorudil** [comment]
voluume please tell us how much volume it should be?

**Pull request closed without merging**